src/EnergyPlus/OutputProcessor.cc

```diff
@@ -5031,7 +4599,8 @@ void ProduceRDDMDD(EnergyPlusData &state)
 
             if (ddVar->ReportedOnDDFile) continue;
 
-            std::string_view timeStepName = timeStepNames[(int)ddVar->timeStepType];
+            static constexpr std::array<std::string_view, (int)TimeStepType::Num> timeStepNamesLocal = {"Zone", "HVAC"};
```

Comment:
The system time step is called `System` everywhere else but it's called `HVAC` in the `.rdd` file end of line comments.